state/common/util/Sentence.kt

```diff
@@ -102,6 +104,9 @@ object Sentence {
                 "transaction that you're connecting in each script you run on TypeDB Server. In 'interactive' mode, you " +
                 "can set on these parameters in the toolbar, and perform queries against the TypeDB server with configured " +
                 "parameters interactively. " + BUTTON_ENABLED_WHEN_CONNECTED
+    const val MANAGE_ADDRESSES_MESSAGE =
+        "Below is the list of addresses for your current cluster connection configuration. You can delete them or " +
+                "add new ones. It is not necessary to input more than two or three addresses."
```

Comment:
"it is not necessary to input more than two or three addresses"; why do we say this?

The user will get the cleanest UX if they input all of their Cluster addresses. I don't think this statement is good to include.

Reply:
I think we should make a statement about a sensible upper limit. It may well be the case in the future that clusters have >10 associated IPs, and inputting every one of them is tedious. 

For now, I've removed it.